site/public/js/autograding-status.js

```diff
@@ -0,0 +1,132 @@
+// Change this variable to change the frequency of the update request, measured in miliseconds
+const refresh_freq = 5000;
+// Change this variable to change the max number of entries in the table before the oldest entries are replaced
+const max_log = 60;
+
+let interval = null;
+
+function updateTable() {
+    clearInterval(interval);
+    $.ajax({
+        // eslint-disable-next-line no-undef
+        url: buildUrl(['autograding_status', 'get_update']),
+        type: 'GET',
+        // eslint-disable-next-line no-undef
+        data: {'csrf_token': csrfToken},
```

Comment:
Rather than telling eslint to ignore these at the top you can put:

/* global buildUrl, csrfToken */

That will tell eslint that it is not in current scope but is global when the browser executes the code.

Also csrfToken is not necessary here as this is a get request. Csrf tokens are only required on post requests.